gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/public/tests/components/privacySettings/js/PrivacySettingsWithPrefsTests.js

```diff
@@ -0,0 +1,51 @@
+/*!
+GPII OAuth2 server
+
+Copyright 2014 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+// Declare dependencies
+/* global document, fluid, gpii, jQuery, jqUnit */
+
+(function ($) {
+    "use strict";
+
+    $(document).ready(function () {
+
+        fluid.registerNamespace("gpii.tests");
+
+        fluid.defaults("gpii.tests.oauth2.privacySettingsWithPrefs", {
+            gradeNames: ["gpii.oauth2.privacySettingsWithPrefs", "autoInit"],
+            model: {
+                user: "testUser"
+            }
+        });
+
+        var assertRenderedText = function (that, root, paths, method) {
```

Comment:
Add these to a global namespace and register them as strings rather than function literals